build.proj

```diff
@@ -77,6 +79,13 @@
     <MSBuild Projects="@(FunctionalTests)" Targets="restore" Properties="TestTargetOS=$(TestOS)netfx" />
   </Target>
 
+  <Target Name="BuildTools" Condition="'$(BuildTools)' == 'true'">
```

Comment:
Since NotSupported.targets are only applicable to AnyOS
```suggestion
  <Target Name="BuildTools" Condition="'$(BuildTools)' == 'true' AND '$(OSGroup)' == 'AnyOS'">
```